routes/entryRoutes.js

```diff
@@ -4,37 +4,64 @@ var routes = {};
 var FormDB = require(path.join(__dirname, '../models/form'));
 
 routes.POSTnewEntry = function(req, res, next) {
+  // This route only handles short term
   var studentID = req.params._id;
+  var period = 'Daily';
   var date = req.body.date.slice(0,10);
   var currentDate = Date.parse(date);
-  var period = req.body.period;
   var attendance = req.body.attendance;
   var behaviorText = req.body.behaviorText;
   var actionSteps = req.body.actionSteps;
   var schoolBehavior= req.body.schoolBehavior;
-  var readingLevels = req.body.readingLevels;
   var teacherFeedback = req.body.teacherFeedback;
   var warnings = req.body.warnings;
   var stars = req.body.stars;
+  var engageContent = parseInt(req.body.engageContent);
```

Comment:
I think I left it there from debugging. It should work without parsing. I can remove it